src/PIL/Image.py

```diff
@@ -1977,7 +2013,7 @@ def putpalette(self, data, rawmode="RGB") -> None:
         self.palette.mode = "RGB"
         self.load()  # install new palette
 
-    def putpixel(self, xy, value):
+    def putpixel(self, xy: tuple[int, int], value: float | tuple[int, ...]) -> None:
```

Comment:
It would appear `value` can also be a list.
https://github.com/python-pillow/Pillow/blob/c7574392b050bf93334ec3c1b1873de2cc3487fb/src/PIL/Image.py#L2035